src/qt/clientmodel.cpp

```diff
@@ -268,3 +269,13 @@ void ClientModel::unsubscribeFromCoreSignals()
     m_handler_notify_block_tip->disconnect();
     m_handler_notify_header_tip->disconnect();
 }
+
+bool ClientModel::getProxyInfo(std::string& ip_port) const
+{
+    proxyType ipv4, ipv6;
+    if (GetProxy((Network) 1, ipv4) && GetProxy((Network) 2, ipv6)) {
```

Comment:
Could call `m_node.getProxy()` here and drop `netbase.h` include above. This would be a little more future-proof with #10102.

Reply:
`netbase.h` is still needed for proxyType, but I can change to `m_node.getProxy`